.github/workflows/build.yml

```
@@ -267,7 +267,7 @@ jobs:
           path: src
 
       - name: Integration test
-        uses: getsentry/github-workflows/sentry-cli/integration-test/@v2
+        uses: getsentry/github-workflows/sentry-cli/integration-test/@0ee6d58acd5ca949358c3f62263a3c592689553e
```

suggestion: update to commit that is also tagged (latest release)
see: https://github.com/getsentry/github-workflows/releases/tag/2.14.1

```suggestion
        uses: getsentry/github-workflows/sentry-cli/integration-test/@a5e409bd5bad4c295201cdcfe862b17c50b29ab7 # v2.14.1
```

We started with an untagged version for a quick fix.

Optional.
If you apply, please also apply the tag comment. I believe our GitHub Actions updater is keeping the comment in sync, too ... see https://github.com/getsentry/sentry-dotnet/pull/4524
If you apply, please also apply to the `- name: Test AOT` step below.

---

Done!